src/app/additionalinquiriesform/AdditionalInquiriesForm.tsx

```diff
@@ -0,0 +1,163 @@
+"use client";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+
+import { Button } from "~/components/ui/button";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+} from "~/components/ui/form";
+import { Input } from "~/components/ui/input";
+import { Textarea } from "~/components/ui/textarea";
+import FormSchema from "./formSchema";
+
+export default function AdditionalInquiriesForm() {
+  const form = useForm<z.infer<typeof FormSchema>>({
+    resolver: zodResolver(FormSchema),
+    defaultValues: {
+      email: "",
+      name: "",
+      message: "",
+    },
+  });
+
+  const {
+    handleSubmit, //
+  } = form;
+
+  const [successMessage, setSuccessMessage] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  async function onSubmit(values: z.infer<typeof FormSchema>) {
+    // eslint-disable-next-line no-console
+    console.log(values);
+
+    try {
+      const response = await fetch("/api/contact", {
```

Comment:
who gave you this API to use?

static page

Reply:
It's just a dummy text